### PR TITLE
chore: improve commitMessage for renovate/renovate docker tags docs update

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Adds `description` field that explains what this is doing in plain English
- Change `commitMessage` to be: `docs: update references to renovate/renovate`
- Use `additionalBranchPrefix` to separate this PR from normal PRs

## Context:

Improves the PR title and commit messages used for PRs like: https://github.com/renovatebot/renovate/pull/8114

This is my first attempt, so please review it thoroughly.
Is there a way I can try this out away from production?
Especially as any mistakes will get auto-merged on the weekly runs...

### CODE REVIEW

**Review by viceice — APPROVED** [review]
LGTM to me, not sure if all things are right. Maybe we should use `additionalBranchPrefix` to seperate from normal renovate updates?

File: base.json [hunk 1]
```diff
@@ -63,11 +63,15 @@
...
+      "description": "Update references to Docker tags for renovate/renovate in Markdown files",
       "packageNames": ["renovate/renovate"],
       "paths": [".md"],
       "extends": ["schedule:weekly"],
       "automerge": true,
-      "stabilityDays": 0
+      "stabilityDays": 0,
+      "commitMessagePrefix": "docs:",
+      "commitMessageAction": "update",
+      "commitMessageTopic": "references to {{{depName}}}"
```
Comment (viceice) [inline base.json]:
```suggestion
      "commitMessageTopic": "references to {{{depName}}}",
      "additionalBranchPrefix": "docs-"
```

**Review by viceice — CHANGES_REQUESTED** [review]

File: base.json [hunk 2]
```diff
@@ -63,11 +63,15 @@
...
     {
+      "description": "Update references to Docker tags for renovate/renovate in Markdown files",
       "packageNames": ["renovate/renovate"],
       "paths": [".md"],
       "extends": ["schedule:weekly"],
       "automerge": true,
-      "stabilityDays": 0
+      "stabilityDays": 0,
+      "commitMessagePrefix": "docs:",
+      "commitMessageAction": "update",
```
Comment (viceice) [inline base.json]:
```suggestion
```

this is the default https://docs.renovatebot.com/configuration-options/#commitmessageaction

File: base.json [hunk 3]
```diff
@@ -63,11 +63,15 @@
...
     },
     {
+      "description": "Update references to Docker tags for renovate/renovate in Markdown files",
       "packageNames": ["renovate/renovate"],
       "paths": [".md"],
       "extends": ["schedule:weekly"],
       "automerge": true,
-      "stabilityDays": 0
+      "stabilityDays": 0,
+      "commitMessagePrefix": "docs:",
```
Comment (HonkingGoose) [inline base.json]: > "commitMessagePrefix": "docs:",

I'm not sure if the `:` is automatically added as a separator between commitMessagePrefix and commitMessageAction by the backend?
Also do I need to add whitespace after the `:`, like this `docs: `?

---

EDIT: resolved by using `semanticCommitScope(docs)` in extends array instead of using `commitMessagePrefix`.

File: base.json (hunk 2 above)
Comment (HonkingGoose) [reply base.json]: Yeah I saw that, but I thought it was better to be explicit. 😄 

If you don't like it, it's gone! 😉

File: base.json [hunk 4]
```diff
@@ -63,11 +63,15 @@
       "groupName": "prettier packages"
     },
     {
+      "description": "Update references to Docker tags for renovate/renovate in Markdown files",
       "packageNames": ["renovate/renovate"],
       "paths": [".md"],
       "extends": ["schedule:weekly"],
```
Comment (viceice) [inline base.json:69]:
```suggestion
      "extends": ["schedule:weekly", ":semanticCommitScope(docs)"],
```

File: base.json (hunk 3 above)
Comment (viceice) [reply base.json]: https://docs.renovatebot.com/presets-default/#semanticcommitscopeltarg0gt

```suggestion
```

**Review by viceice — APPROVED** [review]

File: base.json [hunk 5]
```diff
@@ -63,11 +63,14 @@
       "groupName": "prettier packages"
     },
     {
+      "description": "Update references to Docker tags for renovate/renovate in Markdown files",
       "packageNames": ["renovate/renovate"],
       "paths": [".md"],
-      "extends": ["schedule:weekly"],
+      "extends": ["schedule:weekly", ":semanticCommitScope(docs)"],
```
Comment (rarkins) [inline base.json]: If we want it to be `docs:` then shouldn't it be type=docs and scope=null ?

Comment (viceice) [reply base.json]:
```suggestion
      "extends": ["schedule:weekly"],
```

right 🤦‍♂️

File: base.json [hunk 6]
```diff
@@ -63,11 +63,14 @@
...
       "automerge": true,
-      "stabilityDays": 0
+      "stabilityDays": 0,
+      "commitMessageTopic": "references to {{{depName}}}",
```
Comment (viceice) [inline base.json:72]:
```suggestion
      "commitMessageTopic": "references to {{{depName}}}",
      "semanticCommitType": "docs",
      "semanticCommitScope": null
```

File: base.json [hunk 7]
```diff
@@ -63,11 +63,16 @@
...
+      "description": "Update references to Docker tags for renovate/renovate in Markdown files",
       "packageNames": ["renovate/renovate"],
       "paths": [".md"],
       "extends": ["schedule:weekly"],
       "automerge": true,
-      "stabilityDays": 0
+      "stabilityDays": 0,
+      "commitMessageTopic": "references to {{{depName}}}",
+      "semanticCommitType": "docs",
+      "semanticCommitScope": null
```
Comment (HonkingGoose) [inline base.json]:
```suggestion
      "semanticCommitScope": null,
```

Add missing comma. :smile:

**Review by viceice — APPROVED** [review]

**Review by rarkins — APPROVED** [review]
